_notifications-plugin/index.md

```diff
@@ -0,0 +1,135 @@
+
+The IAM user must have the following permissions to assume a role.
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:Describe*",
+        "iam:ListRoles",
+        "sts:AssumeRole"
+      ],
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+Then add this policy into the IAM user’s trust relationship to actually assume the role.
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+  {
+    "Effect": "Allow",
+    "Principal": {
+    "AWS": "arn:aws:iam::<arn_number>:user/<iam_username>",
+    },
+    "Action": "sts:AssumeRole"
+  }
+  ]
+}
+```
+
+
+## Email as a channel type
+
+To send or receive notifications with emails, choose **Email** as the channel type. Next, select at least one sender and default recipient. To send notifications to more than a few people at a time, select a recipient group. If the Notifications plugin doesn’t currently have the necessary senders or groups, you can add them by first selecting **SMTP sender**, then choose **Create SMTP sender** or **Create recipient group**. Choose **SES sender** to use Amazon Simple Email Service (SES).
+
+### Create email sender
+
+1. Specify a unique name to associate with the sender.
+2. Enter an email address, and, if applicable, its host (for example, smtp.gmail.com), and the port. If you're using SES, enter the IAM role ARN of the AWS account to send notifications from, along with the region.
+3. Choose an encryption method. Most email providers require SSL or TLS, which requires a username and password in the OpenSearch keystore. See [Authenticate sender account](#authenticate-sender-account) to learn more. Selecting an encryption method is only applicable if you're creating an SMTP sender.
+4. Choose **Create** to save the configuration and create the sender. You can create a sender before you add your credentials to the OpenSearch keystore; however, you must [authenticate each sender account](#authenticate-sender-account) before you use the sender in your channel configuration.
+
+### Create email recipient group
+
+1. After choosing **Create recipient group**, enter a unique name to associate with the email group and an optional description.
+2. Select or enter the emails you want to add to the recipient group.
+3. Choose **Create**.
+
+### Authenticate sender account
+
+If your email provider requires SSL or TLS, you must authenticate each sender account before you can send an email. Enter these credentials in the OpenSearch keystore using the CLI. Run the following commands (in your OpenSearch directory) to enter your username and password. The &lt;sender_name&gt; is the name you entered for **Sender** earlier.
+
+```json
+./bin/opensearch-keystore add plugins.alerting.destination.email.<sender_name>.username
+./bin/opensearch-keystore add plugins.alerting.destination.email.<sender_name>.password
```

Comment:
We don't recommend using these settings anymore. Notifications settings for email username and password are:

```
opensearch.notifications.core.email.<sender_name>.username
opensearch.notifications.core.email.<sender_name>.password
```

The Alerting ones you mentioned above are essentially deprecated and will be used as the fallback if the Notifications settings are not set. We're planning on removing the Alerting ones in 3.0 so we should encourage users to stop using them and move over to the Notifications setting namespace.

Reply:
Replaced with the notification settings.